src/waiter-reducer.js

```diff
@@ -46,6 +46,8 @@ const reducerMap = {
       name,
     } = payload
 
+    // If we alredy have a response
+    // we are in refresh mode
     const isRefreshing = !!state.response
```

Comment:
Interesting. I'd never really thought about `isRefreshing`. Thanks for the clarification here.